cli/resolver.rs

```diff
@@ -498,6 +545,23 @@ impl CjsTracker {
     self.get_known_kind_with_is_script(specifier, media_type, None)
   }
 
+  pub fn get_raw_speficier(
+    &self,
+    specifier: &ModuleSpecifier,
+  ) -> Option<Ref<'_, Url, String>> {
+    self.raw_specifier.get(specifier)
+  }
+
+  pub fn set_raw_speficier(
+    &self,
+    specifier: &ModuleSpecifier,
+    raw_specifier: &str,
+  ) {
+    self
+      .raw_specifier
+      .insert(specifier.clone(), raw_specifier.to_string());
+  }
+
```

Comment:
Should we remove the map when the module succeed to `load` as same as below?

https://github.com/denoland/deno/blob/bfc143a5ac233a0db26ff1e29dd7b94a697d862d/cli/cache/parsed_source.rs#L93-L96